.github/workflows/mac-build.yml

```diff
@@ -40,6 +40,8 @@ jobs:
           cp ./LICENSE build
           cp ./CHANGELOG.md build
           cd build
+          install_name_tool -id @rpath/libplay_cpp_sdk.dylib ./lib/libplay_cpp_sdk.dylib
```

Comment:
if we pack @rpath in CI for users, the binary will be used in demo, so we should keep consistent in the setup:
1. https://github.com/cronos-labs/play-cpp-sdk/blob/main/README.md?plain=1#L60 is no longer needed.
2. https://github.com/cronos-labs/play-cpp-sdk/blob/main/README.md?plain=1#L64 make should work.

Reply:
i think libplay_cpp_sdk.dylib should be delivered with @rpath, because it is not system dylib.
it will be invalid in user's envrironment.